src/animations/FadeIn.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { motion, AnimatePresence } from "framer-motion";
+
+type Props = {
+  children: any;
```

Comment:
we should try to avoid `any` whenever possible....This technically should be a `ReactElement` yes?